src/Wolverine/ErrorHandling/PolicyExpression.cs

```diff
@@ -169,6 +169,26 @@ public IAdditionalActions ScheduleRetry(params TimeSpan[] delays)
         return this;
     }
 
+    public IAdditionalActions ScheduleIndefiniteRetry(params TimeSpan[] delays) => ScheduleIndefiniteRetry(CancellationToken.None, delays);
+    
+    public IAdditionalActions ScheduleIndefiniteRetry(CancellationToken cancellationToken, params TimeSpan[] delays)
+    {
+        if (delays.Length == 0)
+        {
+            throw new InvalidOperationException("You must specify at least one delay time");
+        }
+        cancellationToken.ThrowIfCancellationRequested();
+        
+        for (var i = 0; i < delays.Length; i++)
+        {
+            var continuation = new ScheduledRetryContinuation(delays[i]);
+            var slot = i == delays.Length - 1 ? _rule.AddSlot(new IndefiniteScheduledRetryContinuation(continuation, _rule, cancellationToken)) : _rule.AddSlot(continuation);
+            _slots.Add(slot);
```

Comment:
Question to you, @jeremydmiller. If this solution with continuously adding itself as a slot to the rule is an ok solution, would it make sense to add it this _slots list as well? I've had look how this FailureActions thing works, but I don't think that I understand it fully. So there probably is a better way to implement this :)